packages/manager/cypress/e2e/core/parentChild/token-expiration.spec.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  mockAppendFeatureFlags,
+  mockGetFeatureFlagClientstream,
+} from 'support/intercepts/feature-flags';
+import { mockGetLinodes } from 'support/intercepts/linodes';
+import { makeFeatureFlagData } from 'support/util/feature-flags';
+import {
+  accountFactory,
+  accountUserFactory,
+  profileFactory,
+} from 'src/factories';
+import { randomLabel, randomString } from 'support/util/random';
+import {
+  mockGetAccount,
+  mockGetChildAccounts,
+} from 'support/intercepts/account';
+import { mockGetProfile } from 'support/intercepts/profile';
+import { DateTime } from 'luxon';
+import { ui } from 'support/ui';
+
+const mockChildAccount = accountFactory.build({
+  company: 'Partner Company',
+});
+
+const mockChildAccountProxyUser = accountUserFactory.build({
+  username: randomLabel(),
+  user_type: 'proxy',
+});
+
+const mockChildAccountProfile = profileFactory.build({
```

Comment:
```suggestion
const mockChildAccountProxyProfile = profileFactory.build({
```
This feels like it might be slightly more clear?